SampleServer.cpp

```diff
@@ -1,3 +1,18 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Can this be added later in a separate pull request in all files?

Reply:
Yes, dropped.